docs/streams/developer-guide.html

```diff
@@ -2309,6 +2309,25 @@ <h4><a id="streams_developer-guide_interactive-queries_local-window-stores" href
             System.out.println("Count of 'world' @ time " + windowTimestamp + " is " + next.value);
           }
           iterator.close();
+
+	 //Fetch values for all of the windows available within a set time range.
```

Comment:
Nit:  fix indention to align with code from above (some more lines below, too)
NIt: insert space `// Fetch...` (same in below comment)